app/lib/utils/translations/translations_transaction.dart

```diff
@@ -0,0 +1,132 @@
+  String get attestNotificationTitle;
+  String get attestNotificationBody;
+  String get claimRewardsNotificationTitle;
+  String get claimRewardsNotificationBody;
+  String get balanceTransferNotificationTitle;
+  String get balanceTransferNotificationBody;
+}
+
+class TranslationsEnTransaction implements TranslationsTransaction {
+  @override
+  String get endorseNewcomerNotificationBody => 'Thanks for endorsing the newbie!';
+  @override
+  String get endorseNewcomerNotificationTitle => 'Newbie endorsed';
+  @override
+  String get registerParticipantNotificationBody => 'You will receive a reminder one day before.';
+  @override
+  String get registerParticipantNotificationTitle => 'Registered for the next cycle!';
+  @override
+  String get attestNotificationBody =>
+      'If all participants have sent the attestations, you can try to claim the income.';
+  @override
+  String get attestNotificationTitle => 'Attested attendees';
+  @override
+  String get claimRewardsNotificationBody => 'You have already received your community income!';
+  @override
+  String get claimRewardsNotificationTitle => 'Claimed the community income';
+  @override
+  String get balanceTransferNotificationBody => 'The recipient has received the tokens.';
+  @override
+  String get balanceTransferNotificationTitle => 'Transaction completed';
+  @override
+  String get unregisterParticipantNotificationBody =>
+      'Your registration for the next cycle has been cancelled. If you change your mind, you can register again.';
+  @override
+  String get unregisterParticipantNotificationTitle => 'Registration Cancelled';
+}
+
+class TranslationsDeTransaction implements TranslationsTransaction {
+  @override
+  String get endorseNewcomerNotificationBody => 'Danke für das Endorsen des Newbies!';
+  @override
+  String get endorseNewcomerNotificationTitle => 'Newbie Endorsed';
+  @override
+  String get registerParticipantNotificationBody => 'Du erhältst eine Erinnerung einen Tag zuvor.';
+  @override
+  String get registerParticipantNotificationTitle => 'Für den nächsten Cycle angemeldet!';
+  @override
+  String get attestNotificationBody =>
+      'Wenn alle Teilnehmer die Attestierungen geschickt haben, kannst du versuchen das Einkommen zu erhalten.';
+  @override
+  String get attestNotificationTitle => 'Teilnehmer attestiert';
+  @override
+  String get claimRewardsNotificationBody => 'Du hast dein Gemeinschaftseinkommen bereits erhalten.';
+  @override
+  String get claimRewardsNotificationTitle => 'Gemeinschaftseinkommen angefordert';
+  @override
+  String get balanceTransferNotificationBody => 'Der Empfänger hat die Tokens erhalten.';
+  @override
+  String get balanceTransferNotificationTitle => 'Transaktion abgeschlossen';
+  @override
+  String get unregisterParticipantNotificationBody =>
+      'Ihre Anmeldung für den nächsten Zyklus wurde storniert. Wenn Sie es sich anders überlegen, können Sie sich erneut anmelden.';
```

Comment:
Haha, the translations always put the polite form, which we don't use in the App.

```suggestion
      'Du hast dich abgemeldet vom nächsten Cycle. Du kannst dich wider anmelden, wenn du es dir anders überlegst.';
```